fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsAutoCollector.java

```diff
@@ -97,36 +97,54 @@ public void analyzeDb(DatabaseIf<TableIf> databaseIf) {
         }
     }
 
-    public List<AnalysisInfo> constructAnalysisInfo(DatabaseIf<? extends TableIf> db) {
+    protected List<AnalysisInfo> constructAnalysisInfo(DatabaseIf<? extends TableIf> db) {
         List<AnalysisInfo> analysisInfos = new ArrayList<>();
         for (TableIf table : db.getTables()) {
-            if (table instanceof View) {
+            if (table instanceof View || skip(table)) {
```

Comment:
I suggest to put the check `table instanceof View` into `skip` too,
And better only list the supported table type, not `non-supported` type,
In case we may add some new type of table in future and miss to modify here